Cargo.toml

```diff
@@ -33,12 +33,17 @@ section = "rust"
 doc = false
 name = "svd2rust"
 path = "src/main.rs"
+required-features = ["bin"]
+
+[features]
+default = ["bin"]
+bin = ["clap", "clap_conf", "env_logger"]
```

Comment:
Could we maybe use the new `dep:` syntax introduced in 1.60?

You'll have to bump msrv in [README](https://github.com/rust-embedded/svd2rust#minimum-supported-rust-version-msrv) and [in ci](https://github.com/rust-embedded/svd2rust/blob/298f1f933f516302a9596c53ba89e972cda2a343/.github/workflows/ci.yml#L72)

our msrv policy is to bump when needed.

Reply:
> Does this constitute a breaking change?

I think it's not. But if we will use 1.60 feature it technically could be.

Should I doing so?

Reply:
Yes, but it also disables the extra features the optionals make.

without `dep:crate`, you'll have a new feature `crate`

Reply:
Makes sense, will do then.